packages/nimble-tokens/dist/fonts/README.md

```diff
@@ -0,0 +1,16 @@
+# Google Web Fonts
+
+Original font sources:
+
+* [Source Sans Pro](https://fonts.google.com/specimen/Source+Sans+Pro)
+* [Noto Serif](https://fonts.google.com/noto/specimen/Noto+Serif)
+
+Converted to WOFF2 format using [a tff to woff2](https://everythingfonts.com/ttf-to-woff2) recommended by [google-webfonts-helper](https://github.com/majodev/google-webfonts-helper)
+
+# Adding new fonts
+
+New font assets should be added to the `nimble-tokens/dist/fonts/assets` directory and referenced in from `nimble-tokens/dist/fonts/css/fonts.css` so that they will be loaded by applications using nimble.
+
+## Fallback fonts
+
+When adding new fonts, you should also create size-adjusted fallback fonts to use with them. Follow the pattern being used in `nimble-tokens/dist/fonts/css/fonts.css` to define new `@font-face`s that utilize some combination of the descriptors `size-adjust`, `ascent-override`, `descent-override`, and `line-gap-override`. There is an [online tool](https://deploy-preview-15--upbeat-shirley-608546.netlify.app/perfect-ish-font-fallback/) you can use to automatically select values for these descriptors. You should verify that the values produce a good result, and tweak them as needed.
```

Comment:
Should we reference the published version of the tool instead of the preview builds: https://www.industrialempathy.com/perfect-ish-font-fallback/

The published version of the tool no longer configures the ascent-override, generates different calculated values from above, and has the following comment:

> Note, that there are additional related CSS properties [ascent-override](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/ascent-override) and [descent-override](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/descent-override). However, you do not need to worry about these if you set a line-height of any non-default value for all text.

It looks like we have line height tokens but doesn't seem like we use them consistently. Can you comment on the above and include relevant notes in the docs about why or why not it links to the non-published page.

Reply:
We do consistently specify a line-height because it's baked into the definitions of our composed `*-font` tokens.

The published version of the tool gives the same `size-adjust` values as the pre-release one, but leaves off the `ascent-override` value. I can't tell the difference in behavior with or without the `ascent-override`, so I suppose we can leave them off and advocate using the newer version of the tool.